presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogFilter.java

```diff
@@ -20,7 +20,6 @@
 import java.nio.file.Files;
```

Comment:
Ideally, you should divide up this commit ("Fix logs API...") to have one commit with fixing the logs API and one with adding the logs list. (It seems like there are a couple of other assorted bug fixes, mostly around whether paths exist -- those should be either in a separate commit or at least mentioned in the commit message)

You don't need to change this now, though, just keep it in mind going into the future.